.changeset/many-worms-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/consul-ui-toolkit": minor
```

Comment:
I'm not super familiar with changeset, do we want both a minor and major changeset here?

Reply:
Yep it will end up combining them under one release and list all the major/minor/patch changes under corresponding headings in the changelog!